### PR TITLE
Wordcloud for chat

### PR DESCRIPTION
**Pull request by apurvaasf**
Wordcloud for whatsappchat

### CODE REVIEW

File: Whatsapp_Analyser-1.0.ipynb [hunk 1]
```diff
@@ -739,6 +713,86 @@
...
+    "newfile = open('newfile.txt','w')\n",
+    "for item in words:\n",
+    "  newfile.write(\"%s\\n\" % item)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
```
Comment (devashishd12) [inline Whatsapp_Analyser-1.0.ipynb]: Could you please clear the output here?

**Comment by devashishd12** [comment]
Thanks for the PR @apurvaasf!

**Comment by apurvaasf** [comment]
Land it already @dsquareindia.

**Review by devashishd12 — APPROVED** [review]

**Comment by devashishd12** [comment]
Thanks @apurvaasf and congratulations on the merge! 🥳 🎉

**Pull request merged**